src/app/tests/suites/certification/Test_TC_OVENOPSTATE_2_4.yaml

```diff
@@ -0,0 +1,94 @@
+
+
+          [1692859386.707176][6658:6660] CHIP:EM: Rxd Ack; Removing MessageCounter:212756536 from Retrans Table on exchange 36888i
+          [1692859386.707295][6658:6660] CHIP:DMG: ReportDataMessage =
+          [1692859386.707360][6658:6660] CHIP:DMG: {
+          [1692859386.707416][6658:6660] CHIP:DMG:         SubscriptionId = 0x97bb846c,
+          [1692859386.707477][6658:6660] CHIP:DMG:         InteractionModelRevision = 10
+          [1692859386.707533][6658:6660] CHIP:DMG: }
+          [1692859386.707621][6658:6660] CHIP:DMG: MoveToState ReadClient[0xffff80011a90]: Moving to [AwaitingSu]
+          [1692859386.707815][6658:6660] CHIP:EM: <<< [E:36888i S:54523 M:212756537 (Ack:12760634)] (S) Msg TX to 1:0000000000000001 [853D] --- Type 0001:01 (IM:StatusResponse)
+          [1692859386.707901][6658:6660] CHIP:IN: (S) Sending msg 212756537 on secure session with LSID: 54523
+          [1692859386.708167][6658:6660] CHIP:DL: HandlePlatformSpecificBLEEvent 32793
+          [1692859386.708237][6658:6660] CHIP:DL: HandlePlatformSpecificBLEEvent 32793
+          [1692859386.749217][6658:6660] CHIP:EM: >>> [E:36888i S:54523 M:12760635 (Ack:212756537)] (S) Msg RX from 1:0000000000000001 [853D] --- Type 0001:04 (IM:SubscribeResponse)
+          [1692859386.749285][6658:6660] CHIP:EM: Found matching exchange: 36888i, Delegate: 0xffff80011aa0
+          [1692859386.749350][6658:6660] CHIP:EM: Rxd Ack; Removing MessageCounter:212756537 from Retrans Table on exchange 36888i
+          [1692859386.749417][6658:6660] CHIP:DMG: SubscribeResponse is received
+          [1692859386.749489][6658:6660] CHIP:DMG: SubscribeResponseMessage =
+          [1692859386.749538][6658:6660] CHIP:DMG: {
+          [1692859386.749584][6658:6660] CHIP:DMG:         SubscriptionId = 0x97bb846c,
+          [1692859386.749637][6658:6660] CHIP:DMG:         MaxInterval = 0x6,
+          [1692859386.749687][6658:6660] CHIP:DMG:         InteractionModelRevision = 10
+          [1692859386.749734][6658:6660] CHIP:DMG: }
+          [1692859386.749787][6658:6660] CHIP:DMG: Subscription established with SubscriptionID = 0x97bb846c MinInterval = 5s MaxInterval = 6s Peer = 01:0000000000000001
+          [1692859386.749843][6658:6660] CHIP:DMG: MoveToState ReadClient[0xffff80011a90]: Moving to [Subscripti]
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      arguments:
+          values:
+              - name: "message"
+                value: "Enter 'y' after success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "Step 4: TH reads from the DUT the OperationalState attribute"
```

Comment:
From the comment on the other PR - the reason I was confused about this was because the step above indicates you need to verify the event is generated, but the log just shows the subscription being established. No event is received. I thought this was intended to read the event, but actually it's the command above that's weird because the logs don't let you do what you asked the tester to do.

Reply:
I see. just fixed the logs to be verified